sdk/spring/azure-spring-cloud-context/README.md

```diff
@@ -10,12 +10,11 @@ This package helps developers to finish the auto-configuration of Azure Context.
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-context;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-context;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-context</artifactId>
-    <version>2.0.0-beta.1</version>
```

Comment:
We should keep the version here. Then the script will ensure groupid/artifactid/version is align with the tag.